lib/twine/formatters/gettext.rb

```diff
@@ -18,10 +18,8 @@ def default_file_name
       def determine_language_given_path(path)
         path_arr = path.split(File::SEPARATOR)
         path_arr.each do |segment|
-          match = /(..)\.po$/.match(segment)
-          if match
-            return match[1]
-          end
+          match = /(.*)\.po$/.match(segment)
```

Comment:
Works for now, but matching `.*` can be...well, anything ;-) Maybe use a regex similar to Andorid `([a-z]{2}(-[a-z]{2})?)`?